demo/src/components/Cart/Cart.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {Container, Col, Row, Form} from 'react-bootstrap';
+import './Cart.css';
+import {setQuantity} from '../../store/StoreHelpers';
+import Image from 'react-bootstrap/Image';
+import PropTypes from 'prop-types';
+
+/**
+ * Creates a component describing a shopping Cart.
+ * @param {!Object} state The site state
+ * @param {function} setQuantity function to modify quantity of items
+ *      in the global state
+ * @return {!JSX} The component.
+ */
```

Comment:
I guess it's just me not being very familiar with react but why do you need the name here instead of 
`const CartBase = function ({items, setQuantity}) {`

Reply:
That was a mistake, removed it